user-data-store-rest-client/pom.xml

```
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+~ User Data Store
+~ Copyright (C) 2024 Wultra s.r.o.
+~
+~ This program is free software: you can redistribute it and/or modify
+~ it under the terms of the GNU Affero General Public License as published
+~ by the Free Software Foundation, either version 3 of the License, or
+~ (at your option) any later version.
+~
+~ This program is distributed in the hope that it will be useful,
+~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+~ GNU Affero General Public License for more details.
+~
+~ You should have received a copy of the GNU Affero General Public License
+~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.wultra.security</groupId>
+        <artifactId>user-data-store</artifactId>
+        <version>1.3.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>user-data-store-rest-client</artifactId>
+    <name>user-data-store-rest-client</name>
+    <description>User Data Store REST Service Client</description>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.getlime.core</groupId>
+            <artifactId>rest-model-base</artifactId>
+            <version>${wultra-core.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.getlime.core</groupId>
+            <artifactId>rest-client-base</artifactId>
+            <version>${wultra-core.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.wultra.security</groupId>
+            <artifactId>user-data-store-client-model</artifactId>
+            <version>1.3.0-SNAPSHOT</version>
```

Let's use `${project.version}`, please.
```suggestion
            <version>${project.version}</version>
```

---

Ah, yes.